CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+- Insecure, sloppy or any undesirable systems administration practices such as:
+  - Executing remote code by `curl`-ing an untrusted script and piping it to `sudo bash`: `curl https://example.com/awesomescript.sh | sudo bash`
+  - Installing additional bloat within a container
+
+
+We want to encourage the usage of Devops tooling such as config management, containerisation and automation. This will improve the ease of use for all and increase code reuse amongst the community ultimately leading to an easier time for all.
+
+## Directory Layout:
+
+The `docs` folder in the root of the repository is where wiki content should be placed.
+
+The following example directory layout should be adhered to when contributing:
+
+`docs/category/application/index.md`
+
+Additional, more specific articles relating to that same topic should be named differently:
+
+`docs/category/application/specific-topic.md` 
+
+For example, if I wanted to add a brand new, generic article for Home Assistant, my layout would be this:
+
+`docs/home-automation/home-assistant/index.md`
+
+
+For an additional article that is more specific I could name it like so:
+
+`docs/home-automation/home-assistant/docker-deployment.md`
+
+
+Any images embedded in an article should be placed within a folder called `images` within the same directory:
+
+`docs/home-automation/home-assistant/images/example.png`
+
+
+Any other relevant resources (scripts, snippets, etc.) pertaining to the application should be placed within the same folder the relevant application is using.
+
+
+## Build locally
+
+- Clone the repository and `cd` into the directory:
+
+```bash
+git clone https://github.com/selfhostedshow/wiki ~/wiki &&\
+	cd ~/wiki
+```
+
+- Make changes as necessary.
+
+- Start development server on http://localhost:8000:
+
+```bash
+docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material
+```
+
+- In your browser go to http://localhost:8000.
+
+
+## How to Contribute
+
+Please at least try to understand the objectives of this wiki and the organizational layout of the repository before attempting to contribute. 
+
+This Wiki is intended to be for _anyone_ -- regardless of expertise -- who is interested in self-hosting. We encourage anyone interested in contributing to follow these simple steps:
+
+- Fork this repository in Github's web interface by pressing the fork icon..
+- Clone your fork and `cd` into the new directory: `git clone https://github.com/username/wiki && cd wiki`
+- Check out the "staging" branch: `git checkout -b staging`
+- Make changes as desired to the files in the repository on your machine.
+- Add the changed files to your commit and commit the changes to your fork: `git add file1.md file2.md && git commit -m 'updated x,y,z'`
+- Push your commit to your fork: `git push origin staging`
+- Follow the build instructions above (without cloning your fork again). Check that your added content renders properly in your local build.
```

Comment:
Not sure how this statement fits in the list. Is the intention to show what to do to add further contributions?

Reply:
@RealOrangeOne to be more specific are you talking about line 83? Which statement?

Reply:
This statement:

> Follow the build instructions above (without cloning your fork again). Check that your added content renders properly in your local build.

Reply:
@RealOrangeOne, with 32bd7e0d83 I changed the wording to this:
> [Build the wiki locally](https://github.com/selfhostedshow/wiki/blob/master/CONTRIBUTING.md#build-locally) and check that your added content renders properly in your local build.

Does that flow better?

Reply:
My idea is that after someone makes a bunch of changes to the wiki, they should build it locally on their own machine to see what it looks like before submitting a pull request. 

Do you think that's too much? Should I just remove that line altogether?